index.html

```diff
@@ -11,9 +12,97 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
+<style>
+@import url('https://fonts.googleapis.com/css?family=ZCOOL+KuaiLe');
+</style>
 </head>
 
 <body>
-  <h1>My Resume</h1>
+
+	<div class="pageContainer">
+		<header>
+			<nav class="flexNav">
+				<a href="#">Home</a>
+				<a href="#">About</a>
+				<a href="#">Portfolio</a>
+				<a href="#">Contact</a>
+			</nav>
+		</header>
+
+		<main class="heroSection">
+			<img src="img/placeimg_250_300_people.jpg" alt="User photo">
+
+			<h1>Mystery Person - Super Hacker</h1>
+			<p>Hacking is my hobby</p>
+			<div class="line"></div>
+			<p>I already have the job according to the database</p>
+			<button class="heroSection__btn">Meet up!</button>
+		</main>
+
+	<section class="aboutMe">
+		<div class="aboutMe__left">
+		<h2>Listen Up...</h2>
+		<p>
+	Dolor quos perferendis fugiat sapiente consectetur! Iste architecto excepturi fugiat quae reprehenderit. Quod enim vel reiciendis ullam animi! Perferendis delectus nemo neque adipisicing laborum? Blanditiis enim soluta delectus eveniet consectetur?
+</p>
+		</div>
+		<div class="aboutMe__right">
+	<h3>Stuff I'm Good At:</h3>
+	<ul class="li-flex-container">
+		<li>Games</li>
+		<li>Sneaking</li>
+		<li>Stealth</li>
+		<li>Quiet Game</li>
+		<li>coding</li>
+	</ul>
+		</div>
+	</section>
+
+	<section class="workHist">
+		<h2>Work History</h2>
+
+		<div class="row-1">
+			<ul>
+				<li>Yrs</li>
+				<li>Company</li>
+				<li>Title</li>
+				<li>Tech Used</li>
+			</ul>
+		</div>
+
+		<div class="row-2">
+			<ul>
+				<li>1</li>
+				<li>Apple</li>
+				<li>Software Engineer</li>
+				<li>Swift</li>
+			</ul>
+		</div>
+
+		<div class="row-3">
+			<ul>
+				<li>2</li>
+				<li>Microsoft</li>
+				<li>Computer Scientist</li>
+				<li>.NET</li>
+			</ul>
+		</div>
+
+		<div class="row-4">
+			<ul>
+				<li>3</li>
+				<li>Amazon</li>
+				<li>Dev Ops</li>
+				<li>AWS</li>
+			</ul>
+		</div>
+	</section>
+
+	</div>
```

Comment:
nice job recreating a table. But there is a better way :P https://www.w3schools.com/html/html_tables.asp

Reply:
wow i feel really dumb. i struggled with that so bad. i thought i was doing something wrong. thanks for sharing that. that is so much easier. need to figure that out